js/table/tablemanager.ts

```diff
@@ -501,6 +502,15 @@ export class TableManager {
 
         this.currentIndex(0);
     }
+    public toggleSkipDeals(value: boolean) {
```

Comment:
This functionality should not be part of the `TableManager`, since TableManager responsible to keep track of the TableView, This functionality should belongs to the TableView, and I suppose that could be implemented using `TableViwe.sitOut()` and `TableView.comeBack()` methods.